docs/send-data/kubernetes/collecting-logs.md

```diff
@@ -504,6 +504,128 @@ sumologic:
+        table:
+          ## send all logs from `production` namespace to `awss3/production` exporter
+          ## as `useDefaultExporters` is set to `true` (default), all logs will be sent to `sumologic` as well
+          - exporter: awss3/production
+            statement: route() where resource.attributes["namespace"] == "production"
+metadata:
+  logs:
+    statefulset:
+      extraEnvVars:
+        # Set env variables for awss3 exporter
+        - name: AWS_ACCESS_KEY_ID
+          value: "< YOUR AWS ACCESS KEY >"
+        - name: AWS_SECRET_ACCESS_KEY
+          value: "< YOUR AWS SECRET ACCESS KEY >"
+```
+
+`sumologic.logs.otelcol.routing.table` is a list of maps, which consist of the keys `exporter` and `statement`.
+
+Logs are sent to `exporter` when `statement` is met. `exporter` should be defined in `sumologic.logs.otelcol.extraExporters`,
+or the default `sumologic` may be used. `statement` has to be written in
+[Open Telemetry Transformation Language](https://github.com/open-telemetry/opentelemetry-collector-contrib/blob/main/processor/routingprocessor/README.md#tech-preview-opentelemetry-transformation-language-statements-as-routing-conditions).
+
+:::note
+Remember to use [attributes after translation to Sumo Logic schema](https://github.com/open-telemetry/opentelemetry-collector-contrib/tree/main/processor/sumologicprocessor#attribute-translation). For xample, use `namespace` instead of `k8s.namespace.name`.
+:::
+
+:::note
+If a log matches multiple statements, it will be sent to all corresponding exporters.
+For example, the following routing table will send logs from the `production` namespace to `sumologic/production` and `awss3/production`:
+
+```yaml
+- exporter: awss3/production
+  statement: route() where resource.attributes["namespace"] == "production"
+- exporter: sumologic/production
+  statement: route() where resource.attributes["namespace"] == "production"
+```
+
+:::
+
+### Send logs selectively to multiple locations
+
+This scenario differs from the previous ones by not sending all logs to Sumo Logic. If you want to send some data to Sumo Logic and some (the same or other) data to other locations, you need to disable default exporters. Set `sumologic.logs.otelcol.useDefaultExporters` to `false,` and then set `sumologic.logs.otelcol.routing.fallbackExporters` to a list of exporters that are going to be used for the logs which do not match any of the routing table statements.
+
+In the following example, logs from the `production` namespace are going to be sent to the production Sumo Logic exporter, and all remaining logs will be sent to the default Sumo Logic exporter:
+
+```yaml
+sumologic:
+  logs:
+    otelcol:
+      ## useDefaultExporters set to false to stop sending all logs to default exporters
+      useDefaultExporters: false
+      extraExporters:
+        ## define `sumologic/production` exporter
+        sumologic/production:
+          endpoint: http://my-production-sumologic-otlp-source
```

Comment:
This, I think we should move to an env variable mounted from a Secret in this configuration. This is what we do internally and it should probably be encouraged as a best practice.

Reply:
Done